docs/UIDS-GIDS.md

```diff
@@ -237,7 +237,8 @@ the artifacts the container manager persistently leaves in the system.
 |                 65535 | 16bit `(uid_t) -1`    | Linux         |                               |
 |          65536…524287 | Unused                |               |                               |
 |     524288…1879048191 | Container UID ranges  | `systemd`     | `nss-mymachines`              |
-| 1879048192…4294967294 | Unused                |               |                               |
+| 1879048191…2147483647 | Unused                |               |                               |
+| 2147483648…4294967294 | HIC SVNT LEONES       |               |                               |
```

Comment:
I have never seen the phrase "HIC SVNT LEONES", and had to Google it. I suggest replacing this with something simpler.

Reply:
But why else did I have latin in school if not for the purpose of showing off in some technical documentation with it?